index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header> 
+         <a>Jhon Neira</a>
+         <nav class="menu-icon">
+            <img src="Images/Menu.png" alt="menu-icon">
+         </nav> 
+    </header>
+    <main>
+        <h1>
+            Hey There. <br>
+            I'm Alejo
+        </h1>
+        <h2>
+            I am a Software Developer
+        </h2>
+        <a>
+            I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+don't hestiate to contact me.
+        </a>
+        <br> 
+        <ul class="Social-container">
+            <li><img class="Gato" src="Images/Gato.png" alt="GitHub"></li>
```

Comment:
- [ ] Each li shoud contain `<a>` tag for address. For ex:-
`      <li>
          <a href="https://twitter.com">
             <i class="fa fa-twitter"></i>
          </a>
        </li>`
- [ ] For Icons always use [fontawesome ](https://fontawesome.com/)website.